astro/src/content/blog/announcing-fusionauth-1-62.mdx

```diff
@@ -0,0 +1,96 @@
+---
+publish_date: 2025-12-18
+title: Announcing FusionAuth 1.62 - The Refined Reindeer
+description: "FusionAuth 1.62.0 arrives with a polished, high-performance Admin UI. This release also features Pre-Verified Identities, smarter MFA Lambdas, and Tenant-Scoped IdPs for better scalability."
+authors: Brad McCarty
+image: /img/blogs/release-1-62/fusionauth-1-62.png
+categories: Product
+tags: SAML, MFA, identity verification, multi-tenant, JWT
+excerpt_separator: "{/* more */}"
+---
+We're excited to release FusionAuth 1.62.0. This version features a refreshed admin UI, the MFA Lambda for intelligent authentication decisions, tenant-scoped identity providers, EdDSA support, identity pre-verification during registration, and performance improvements.
+
+We're calling this release The Refined Reindeer because of our focus on polish and precision.
+
+![The Refined Reindeer](/img/blogs/release-1-62/reindeer.jpg)
+
+## Fresh New Look for the Admin UI
+
+FusionAuth got a makeover. We've refreshed the admin UI with improved spacing, updated colors, and refinements throughout. You'll still be right at home, but everything feels more polished and professional.
+
+![The refreshed Admin App UI](/img/blogs/release-1-62/manage-user-ds.png)
+
+We’ve also made significant performance improvements for deployments with large numbers of applications and tenants. The admin UI stays responsive even with thousands of applications.
+
+## Pre-Verified Identities
+
+Traditional verification happens after an account is created. Users create the account, FusionAuth then sends a verification email, and you hope the user completes it. Maybe there was a typo, or maybe a user is squatting on an identity. Either way, this creates an admin burden of managing unverified accounts, and introduces account takeover risks.
+
+Identity Pre-Verification ensures email addresses or phone numbers are verified during registration, before the account is created.
+
+### How It Works
+
+1. Users provide their info in a registration form
+2. They verify via a code or link sent immediately
+3. No account is created until verification succeeds
+
+This should cut down on the administrative burden of cleaning up “ghost” accounts, and everyone could use a bit less time spent doing “busy work.”
+
+## MFA Lambda: Smarter Authentication Decisions
+
+The MFA Lambda gives you programmatic control over when multi-factor authentication is required during login.
```

Comment:
```suggestion
The MFA Lambda provides programmatic control of multi-factor authentication in your application.
```
[suggestion]: 'during login' isn't quite right, since the MFA lambda can also be used for step-up, correct?